detectron2/modeling/matcher.py

```diff
@@ -2,7 +2,10 @@
 from typing import List
 import torch
 
+from detectron2.layers import nonzero_tuple
 
+
+@torch.jit.script
```

Comment:
can you leave the class not scripted, but add a test in `modeling/test_matcher` that tests for scriptability? It can be like https://github.com/facebookresearch/detectron2/blob/de098423c675dad38c23110407926ccf2919474d/tests/modeling/test_anchor_generator.py#L70-L75 that asserts the outputs are the same.

Reply:
unless there is a significant speed improvement to always enable script mode for this module, we'd like to keep things eager by default for better flexibility during development

Reply:
done:)

Reply:
I come across another problem about jit and link to an pytorch issue in the codes. I'm trying to figure out if there's a more intuitive workaround about it.